### ISSUE TITLE
text-line segmentation

### ISSUE THREAD

**Issue by seekingdeep** [divyanshjoshi/Attention-U-Net-Newspaper-Text-Block-Segmentation#2]
Hi there,
your implementation focuses on segmenting baselines and blocks,
but what about text-line segmentation

**Comment by divyanshjoshi** [comment]
The model is capable of segmenting text-lines too, but you need the corresponding ground truth data to achieve text-line segmentation, the attention U-Net is a general model and can be utilized in different segmentation tasks given the pixel ground truth, hope that helps, thanks!

**Comment by seekingdeep** [comment]
in that case, what should be the training dataset structure
a sample of the dataset would clear things

**Comment by divyanshjoshi** [comment]
The train set mainly consists of images and their corresponding binary masks as ground truth, you can get more clarity on this once the paper has been indexed

**Comment by seekingdeep** [comment]
alright
